lib/bolt/transport/ssh/connection.rb

```diff
@@ -308,51 +240,17 @@ def write_remote_file(source, destination)
         rescue StandardError => e
           raise Bolt::Node::FileError.new(e.message, 'WRITE_ERROR')
         end
-
-        def make_tempdir
-          tmpdir = target.options.fetch('tmpdir', '/tmp')
-          tmppath = "#{tmpdir}/#{SecureRandom.uuid}"
-          command = ['mkdir', '-m', 700, tmppath]
-
-          result = execute(command)
-          if result.exit_code != 0
-            raise Bolt::Node::FileError.new("Could not make tempdir: #{result.stderr.string}", 'TEMPDIR_ERROR')
-          end
-          path = tmppath || result.stdout.string.chomp
-          RemoteTempdir.new(self, path)
-        end
-
-        # A helper to create and delete a tempdir on the remote system. Yields the
-        # directory name.
-        def with_remote_tempdir
-          dir = make_tempdir
-          yield dir
-        ensure
-          dir&.delete
-        end
-
-        def write_remote_executable(dir, file, filename = nil)
-          filename ||= File.basename(file)
-          remote_path = File.join(dir.to_s, filename)
-          write_remote_file(file, remote_path)
-          make_executable(remote_path)
-          remote_path
-        end
+        # We use `copy_file` for compatibility with the local transport in
```

Comment:
I think we should settle on one name. IMO aliases are for supporting public APIs otherwise they just make things more confusing.